src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -125,8 +133,8 @@ class InvalidArgumentValueError(UserFault):
     pass
 
 
-class ArgumentParseError(UserFault):
-    """ Fallback of the argument parsing related errors.
+class ArgumentUsageError(UserFault):
```

Comment:
ArgumentParseError seems making more sense.

Reply:
Want use this type to cover the original UsageError, which usually are the across arguement error